wire-ios-sync-engine/Source/Calling/CallCenterSupport.swift

```diff
@@ -43,6 +43,19 @@ public enum AVSConversationType: Int32 {
 
 }
 
+extension AVSConversationType {
```

Comment:
Suggestion: move this extension to a file `AVSConversationType+Conference.swift`. It makes it easier to discover in the project and the files smaller.